graylog2-web-interface/src/components/common/mode-pipeline.js

```diff
@@ -0,0 +1,103 @@
+import URLUtils from 'util/URLUtils';
```

Comment:
I think the file name could be more descriptive or even show an indication that this file is related to `SourceCodeEditor`. What do you think about renaming it or putting it in a folder where it's clearer what it is?